addons/l10n_sa/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="l10n_sa_use_additional_footer" model="ir.config_parameter">
```

Comment:
Remove this (and its use).
It was not asked by Yatin and it's not something that people would know about (and so will be useless).
Else, we would need to have the same king of logic for every bit we add in the report invoice.

If someone would really want to avoid having it, they could inherit the view to remove its use. It would be as obscure/dev needed.